rudder-web/src/main/scala/bootstrap/liftweb/AppConfig.scala

```diff
@@ -397,6 +397,11 @@ object RudderConfig extends Loggable {
   val historizationRepository : HistorizationRepository =  historizationJdbcRepository
   val roApiAccountRepository : RoApiAccountRepository = roLDAPApiAccountRepository
   val woApiAccountRepository : WoApiAccountRepository = woLDAPApiAccountRepository
+  lazy val dataSourceRepository = new DataSourceRepoImpl(
```

Comment:
this lazy val is in the middle of non lazy val - could you move it with others lazy val ?